eyepop/compute/status.py

```
@@ -46,58 +42,48 @@ async def wait_for_session(
+                    last_message = f"HTTP {response.status}"
+                    log.debug(f"GET /health - status: {response.status} (attempt {attempt})")
                     await asyncio.sleep(interval)
                     continue
 
-                session_response = ComputeApiSessionResponse(**(await response.json()))
+                body = await response.json()
+
+                try:
+                    session_response = ComputeApiSessionResponse(**body)
+                except ValidationError:
+                    log.debug(f"GET /health - status: 200, simple health response (attempt {attempt})")
```

Ruff (0.15.11)

[warning] Logging statement uses f-string

(G004)